package/jabra-direct.nuspec

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+<?xml version='1.0' encoding='UTF-8'?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>jabra-direct</id>
-    <version>4.0.4873</version>
+    <version>4.0.4380</version>
```

Comment:
is that correct? looks like a downgrade...

Reply:
I downgraded it so that the package is built on the CI and the repository is updated automatically.